app/code/Magento/CheckoutAgreements/Api/CheckoutAgreementsListingInterface.php

```diff
@@ -0,0 +1,23 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+namespace Magento\CheckoutAgreements\Api;
+
+/**
+ * Interface CheckoutAgreementsListingInterface
+ *
+ * @api
+ */
+interface CheckoutAgreementsListingInterface
+{
+    /**
+     * Listing of checkout agreements.
+     *
+     * @param \Magento\Framework\Api\SearchCriteriaInterface $searchCriteria
+     * @return \Magento\CheckoutAgreements\Api\Data\AgreementInterface[]
+     */
+    public function getListing(\Magento\Framework\Api\SearchCriteriaInterface $searchCriteria) : array;
```

Comment:
To remain consistent with the rest of the codebase it would make sense to rename method to `getList`